README.md

```diff
@@ -538,6 +546,45 @@ Failure Response : `HTTP_404_NOT_FOUND`
 }
 ```
 
+### Search Customers and Addresses
+
+URL : `http://127.0.0.1:8080/api/customers/<query_field>=<query_value>`
```

Comment:
Add the ? here as well before <query_field>

Reply:
done